tests/test_api.py

```diff
@@ -313,6 +317,7 @@ def test_create_worker_build(self, osbs, inner_template, outer_template,
             'git_ref': TEST_GIT_REF,
             'git_branch': branch,
             'user': TEST_USER,
+            'filesystem_koji_task_id': TEST_FILESYSTEM_KOJI_TASK_ID,
```

Comment:
I think this is the only test that it makes sense to add `filesystem_koji_task_id` to the list of `expected_kwargs`.

Additionally, test_build_request.py could be enhanced to cover changes to `render_add_filesystem`. See if we can leverage the existing test `test_prod_custom_base_image`.